src/Symfony/Component/Config/Resource/ClassExistenceResource.php

```diff
@@ -97,21 +97,24 @@ public function isFresh($timestamp)
     /**
      * @internal
      */
-    public function serialize()
+    public function __sleep(): array
     {
         if (null === $this->exists) {
             $this->isFresh(0);
         }
 
-        return serialize([$this->resource, $this->exists]);
+        $this->serialized = [$this->resource, $this->exists];
+
+        return ['serialized'];
```

Comment:
wouldn't returning `['resource', 'exists']` work just fine here, without the need of adding a new dynamic property ?

Reply:
and this applies to many other classes too, then.